packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.tsx

```diff
@@ -75,6 +75,14 @@ export class GroupedList extends BaseComponent<IGroupedListProps, IGroupedListSt
     }
   }
 
+  public componentWillMount() {
+    const { groupProps } = this.props;
+
+    if (groupProps && groupProps.isAllGroupsCollapsed) {
+      this.toggleCollapseAll(groupProps.isAllGroupsCollapsed);
+    }
+  }
```

Comment:
Should we also do this in `componentWillReceiveProps`?

Reply:
I'm treating `isAllGroupsCollapsed` as a sort of 'initial value', similar to `defaultValue`: https://facebook.github.io/react/docs/uncontrolled-components.html#default-values

If we want to have `isAllGroupsCollapsed` behave as a normal prop (i.e. updating the prop will update the collapsed/expanded rendering of GroupedList, via `componentWillReceiveProps`), then the component rendering a `<GroupedList />` will need to keep track of the collapsed state, which is not something I think we want to do.

Reply:
Should we reflect that in the name? `initialAllGroupsCollapsed` (deprecate the current one, shouldn't affect anyone never had any logic behind it, and add a new one)?

Reply:
Sounds good; I'll make that change

Reply:
Updated!